task_assign/info[] info0_vect
task_assign/info[] info_vect
